### PR TITLE
Create 2016-11-22-sustaining-research-projects.md

### PR DESCRIPTION
**Pull request by danielskatz**
Here's the initial draft of blog on guide to sustainability models for research software projects, as discussed in yesterday meeting.

I'm not sure if the @twitter-handle style is right for the blog - I could also change to just naming people if that's better.

### CODE REVIEW

File: _posts/2016-11-22-sustaining-research-projects.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+name: sustaining-research-projects.md
```
Comment (sckott) [inline _posts/2016-11-22-sustaining-research-projects.md]: don't put the file extension in the `name` slot

File: _posts/2016-11-22-sustaining-research-projects.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+layout: post
+title: "A guide to sustainability models for research software projects"
+date: 2016-11-22
+authors:
+  - name: Daniel S. Katz
+    url: https://github.com/danielskatz/sustaining-research-projects
```
Comment (sckott) [inline _posts/2016-11-22-sustaining-research-projects.md]: usually we link to a page about the person, not a specific project

File: _posts/2016-11-22-sustaining-research-projects.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+(https://github.com/nayafia/lemonade-stand), I wanted to understand how
+the projects themselves are sustained. @nayafia thought this was out of scope for her document
+(see https://github.com/nayafia/lemonade-stand/issues/1), so I forked her work and created
+
+  https://github.com/danielskatz/sustaining-research-projects
+
+My intent has been to list how projects support themselves over time. Each funding category
+ideally links to several real examples, and when possible, to a useful article or page instead of just a homepage.
+
+The categories are not mutually exclusive. For example, a project might have a foundation but also use
```
Comment (sckott) [inline _posts/2016-11-22-sustaining-research-projects.md]: by _have a foundation_, do you mean get funding from a foundation?

Comment (danielskatz) [inline _posts/2016-11-22-sustaining-research-projects.md]: I actually meant that the project might be part of a non-profit entity (foundation) that enables it to accept funds (e.g. apache projects within the Apache Foundation)

Comment (danielskatz) [inline _posts/2016-11-22-sustaining-research-projects.md]: > I wonder if you could do a vignette of an organization to give a flavor of what one organization does.

I don't think there's a simple vignette that would be useful. Right now, most organizations only do one or two things.

Comment (sckott) [inline _posts/2016-11-22-sustaining-research-projects.md]: > meant that the project might be part of a non-profit entity

makes sense

got it on vignette

**Comment by stefaniebutland** [comment]
@danielskatz I think it's better to refer to people by name in this post, but you could link to their github to connect to their online presence and their work if you want

**Comment by danielskatz** [comment]
> I think it's better to refer to people by name in this post, but you could link to their github to connect to their online presence and their work if you want

now fixed

**Comment by stefaniebutland** [comment]
@danielskatz Just a reminder that this is scheduled to be posted week of Jan 9th. I don't need to be involved further (and will be busy in DC all next week). @sckott will post as usual once you give him the go-ahead.

**Comment by danielskatz** [comment]
I just re-read it and caught one more thing, but I think it is now ready to go next week.  Thanks @stefaniebutland and @sckott !

**Comment by karthik** [comment]
Post looks good to me. cc @stefaniebutland

**Pull request merged**